packages/core/src/checkbox/Checkbox.css

```diff
@@ -18,6 +18,10 @@
   cursor: var(--salt-selectable-cursor-readonly);
 }
 
+.saltSwitch-label {
+  padding-top: var(--salt-spacing-25);
+}
```

Comment:
is this Switch styling necessary here?

Reply:
the styling is to center the text with the switch - https://github.com/jpmorganchase/salt-ds/issues/2372